addons/digest/models/digest.py

```diff
@@ -146,7 +146,7 @@ def _action_send_to_user(self, user, tips_count=1, consum_tips=True):
         # create a mail_mail based on values, without attachments
         mail_values = {
             'auto_delete': True,
-            'email_from': self.company_id.partner_id.email_formatted if self.company_id else self.env.user.email_formatted,
+            'email_from': self.company_id.partner_id.email_formatted or self.env.user.email_formatted or self.env.ref('base.user_root').email_formatted,
```

Comment:
in practice the CRON will be executed by Odoobot, not by the admin

so in your commit, this sentence is false, it will be the odoobot email in most cases (except if the admin run the CRON manually)
> Note that only the admins have rights for the digest, and so the "From" will be the admin's email in such cases.

Can you split lines like this ? (I have to scroll horizontally to read everything on my screen)

```suggestion
            'email_from': (
                self.company_id.partner_id.email_formatted
                or self.env.user.email_formatted
                or self.env.ref('base.user_root').email_formatted
            ),
```